### ISSUE TITLE
Email Error

### ISSUE THREAD

**Issue by JoaoOlsenTecnico** [magento/magento2#23398]
Summary
When the user creates an account in my store, the "Welcome" email does not have the sender's name and transactional emails, such as purchase emails, correctly contain the sender's name defined in "E- mail address store ". I used Bitnami to install my Magento 2.3.0 and it is allocating on an AWS Ubuntu machine, I also use the MagePal Mail Module to connect to the mail server, but when I contacted them, I was informed that they did not is a module error. I'm also using a theme, but I do not think it will interfere with that part.

Examples
Here are two examples:

![image](https://user-images.githubusercontent.com/32138879/60106666-fd4c2000-973b-11e9-8ffe-26ab3263b2f6.png)
Translation:
"Atualização do seu pedido de Megalentes"
equals:
"Update your request for Megalentes"
(is a transactional e-mail)

![image](https://user-images.githubusercontent.com/32138879/60106633-eefe0400-973b-11e9-97e7-7c2da2c23c06.png)
![image](https://user-images.githubusercontent.com/32138879/60107298-0a1d4380-973d-11e9-8d65-165605d28548.png)

Translation:
"Bem-vindo a Megalentes"
equals:
"Welcome to Megalentes"
(It's a welcome email)

Proposed solution
I have analyzed the file "SenderResolver.php" and the function "public function resolves ($ sender, $ scopeId = null)" when I changed the code as follows

```
         if (! is_array ($ sender)) {
             $ result ['name'] = $ this -> _ scopeConfig-> getValue (
                 'trans_email / ident_'. $ sender. '/ name',
                 \ Magento \ Store \ Model \ ScopeInterface :: SCOPE_STORE,
                 $ scopeId
             );
             $ result ['email'] = $ this -> _ scopeConfig-> getValue (
                 'trans_email / ident_'. $ sender. '/email',
                 \ Magento \ Store \ Model \ ScopeInterface :: SCOPE_STORE,
                 $ scopeId
             );
         } else {
             $ result = null;
         }

```
I no longer received the welcome emails with the empty sender name but received the emails that continue the sender's name, that is, when the method enters the "if" it correctly sends the sender's name, already when it falls in the " else "it sends without the sender's name, what can it be?

**Comment by JoaoOlsenTecnico** [comment]
@magento-devops some news??

**Comment by gwharton** [comment]
Fixed in 2.3-develop

**Test case**

```
Stores -> All Stores
    Create Website
        Name = Website1
        Code = website1
    Create Website
        Name = Website2
        Code = website2
    Create Store
        Website = Website1
        Name = Store1
        Code = store1
        Root Category = Default Category
    Create Store
        Website = Website2
        Name = Store2
        Code = store2
        Root Category = Default Category
    Create Store View
        Store = Store1
        Name = View1
        Code = view1
        Status = Enabled
    Create Store View
        Store = Store2
        Name = View2
        Code = view2
        Status = Enabled
Change Scope to Website 1
    Stores -> Configuration -> Store Email Addresses
        General Contact
            Name = ownerwebsite1
            Email = ownerwebsite1@domain.com
Change Scope to Website 2
    Stores -> Configuration -> Store Email Addresses
        General Contact
            Name = ownerwebsite2
            Email = ownerwebsite2@domain.com
Customers -> All Customers -> Add new Customer
    Associate to Website = Website1
    First Name = test
    Second Name = user
    Email = testuser@domain.com
    Send Welcome email from = View1
Customers -> All Customers -> Add new Customer
    Associate to Website = Website2
    First Name = test2
    Second Name = user2
    Email = test2user2@domain.com
    Send Welcome email from = View2

Verify Email 1 received from ownerwebsite1 <ownerwebsite1@domain.com>
Verify Email 2 received from ownerwebsite2 <ownerwebsite2@domain.com>
```

**2.3.0 (ERROR)**
Email 1 = From: ownerwebsite1@domain.com
Email 2 = From: ownerwebsite2@domain.com
    
**2.3-develop (FIXED)**
Email 1 = From: =?utf-8?Q?ownerwebsite1?= <ownerwebsite1@domain.com>
Email 2 = From: =?utf-8?Q?ownerwebsite2?= <ownerwebsite2@domain.com>

I think the fix for this was in 2.3.1 so updating should resolve your issue.
